client/app/states/login/login.state.js

```diff
@@ -74,8 +74,11 @@ function StateController (exception, $state, Text, RBAC, API_LOGIN, API_PASSWORD
         Session.destroy()
       }
     })
-    .catch(() => {
-      exception.catch('Login failed, possibly invalid credentials.')
+    .catch((response) => {
+      if (response.status === 401) {
+        vm.credentials.password = ''
```

Comment:
should `vm.credentials.login` also be cleared out?

Reply:
I realize ref was for only password to be blank, but if it was an unsuccessful login attempt :-/

Reply:
The RFE just requested password be cleared out.